api/modules/dataSyncs.js

```diff
@@ -9,10 +13,30 @@ const db = require('../models')
 const invoicelyCodebase = require('../scripts/invoicelyCodebase')
 const timeLogging = require('../scripts/timeLogging')
 const { INVOICELY_CSV_PATH } = require('../config/constants')
-const { GITHUB, TOGGL } = require('../config/credentials')
+const { GITHUB } = require('../config/credentials')
 
 const dataSyncs = module.exports = (() => {
 
+    const syncContributions = async (github_url, user_url, author, assignee, matchingIssue, issue) => {
```

Comment:
I recommend wrapping this in 1 object so that the order of parameters does not matter